packages/ndla-ui/src/Frontpage/FrontpageProgramMenu.tsx

```diff
@@ -43,20 +43,36 @@ const FrontpageProgramMenu = ({
   t,
 }: Props & tType) => {
   const [showSubjects, setShowSubjects] = useState(false);
+  const isWindowContext = typeof window !== 'undefined';
+
+  useEffect(() => {
+    if (isWindowContext) {
+      const rememberSubjects =
+        window.localStorage.getItem('showSubjects') || '';
+      setShowSubjects(rememberSubjects.localeCompare('true') === 0);
+    }
+  }, []);
+
+  const toggleSubjects = (toggle: boolean) => {
+    setShowSubjects(toggle);
+    if (isWindowContext) {
+      window.localStorage.setItem('showSubjects', `${toggle}`);
```

Comment:
Kanskje greit å prefixe local-storage nøkkelen med f.eks "frontpage"?
```suggestion
      window.localStorage.setItem('frontpageShowSubjects', `${toggle}`);
```

Reply:
Eg tenkte tanken, men denne lagres jo på localstorage for ndla.no og tenkte det var spesifikt nok. Men kan legge på frontpage i tilfelle vi skulle ha behov for å lagre fleire ting i localstorage.